api-admin/src/main/java/bio/terra/pearl/api/admin/service/DataRepoExportExtService.java

```diff
@@ -52,10 +52,9 @@ public List<DataRepoJob> getJobHistoryForDataset(
     Portal portal = authUtilService.authUserToPortal(user, portalShortcode);
     authUtilService.authUserToStudy(user, portalShortcode, studyShortcode);
 
-    StudyEnvironment studyEnv =
-        studyEnvironmentService.findByStudy(studyShortcode, environmentName).get();
+    Dataset dataset = dataRepoExportService.getDatasetByName(datasetName);
```

Comment:
One of my main points of this refactor is to lean more on using `datasetId`s as the lookup key, but that's a concept hidden from the user and `datasetName`s are globally unique in Juniper and TDR anyway so it's alright to do it here